src/main/java/chronosacaria/mcdw/blocks/McdwTempCobwebBlock.java

```diff
@@ -0,0 +1,71 @@
+package chronosacaria.mcdw.blocks;
+
+import chronosacaria.mcdw.api.util.CleanlinessHelper;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.CobwebBlock;
+import net.minecraft.server.world.ServerWorld;
+import net.minecraft.sound.SoundCategory;
+import net.minecraft.sound.SoundEvents;
+import net.minecraft.state.StateManager;
+import net.minecraft.state.property.IntProperty;
+import net.minecraft.state.property.Properties;
+import net.minecraft.state.property.Property;
+import net.minecraft.text.Text;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.util.math.random.Random;
+import net.minecraft.world.World;
+import org.checkerframework.checker.units.qual.A;
+
+
+public class McdwTempCobwebBlock extends CobwebBlock {
+    public static final IntProperty AGE;
+
+    public McdwTempCobwebBlock(Settings settings) {
+        super(settings);
+        settings.dropsNothing();
+        this.setDefaultState((BlockState)((BlockState)this.stateManager.getDefaultState()).with(AGE, 0));
```

Comment:
You appear to have some unnecessary casts here. You should be able to do this.setDefaultState(this.stateManager.getDefaultState().with(AGE, 0));